grails-app/views/stockMovement/show.gsp

```diff
@@ -206,6 +206,14 @@
                                 ${stockMovement?.stocklist?.name?:warehouse.message(code:"default.none.label")}
                             </td>
                         </tr>
+                        <tr class="prop">
```

Comment:
Perhaps it would be good to hide this part if there is no approvers at all (this indicates it is not supported)